packages/dashboard-frontend/src/services/devfileApi/typeguards.ts

```diff
@@ -35,7 +34,6 @@ export function isDevWorkspace(workspace: unknown): workspace is devfileApi.DevW
   return workspace !== undefined
     && isDevWorkspaceLike((workspace as devfileApi.DevWorkspace))
     && (workspace as devfileApi.DevWorkspace).apiVersion !== undefined
-    && isDevWorkspaceMetadata((workspace as devfileApi.DevWorkspace).metadata)
```

Comment:
I wonder if a devworkspace without metadata makes any sense on the dashboard? 
cc @sleshchenko

Reply:
I will return it.

Reply:
> I wonder if a devworkspace without metadata makes any sense on the dashboard?

It makes sense to print ERROR is such devworkspace exists because technically it's not possible from K8s API point of view.

Reply:
I have removed it